### DISCUSSION TITLE
[Feature] ffmpeg utility

### DISCUSSION THREAD

**Discussion by jmiravet** [immich-app/immich#11512] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add a utility that allows to compress videos, maybe based on ffmpeg library. Similar to the deduplicate, allowing to select or unselect videos. Maybe sorting them by file size, and tracking the ones that haven't been compressed before. 

On the server side, creating a setting for max frame size, frame rate and audio channels to convert to.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1202/#5266
